src/styles/searchresults.css

```diff
@@ -0,0 +1,25 @@
+.card-image {
+  border-radius: 20%;
+  width: 100%;
+  height: auto;
+  margin: auto;
```

Comment:
The height property does nothing, it should be removed. Also remember that margins auto property does not work for vertical directions, so this should be replaced with `margin: 0 auto`